src/main/java/nebula/plugin/metrics/MetricsPluginExtension.java

```diff
@@ -121,8 +128,41 @@ public void setSanitizedProperties(List<String> sanitizedProperties) {
         this.sanitizedProperties = checkNotNull(sanitizedProperties);
     }
 
+    public String getSuroLogEventName() {
```

Comment:
Rather than clutter the top level of the extension with a whole bunch of Suro specific properties, we might want a 'suro' container instead?